internal/service/anycast/api_anycast_host_resource_test.go

```diff
@@ -0,0 +1,385 @@
+package anycast_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"net/http"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/infobloxopen/bloxone-go-client/anycast"
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/acctest"
+)
+
+func TestAccOnPremAnycastHostResource_basic(t *testing.T) {
```

Comment:
all tests in this file must be renamed
```suggestion
func TestAccAnycastHostResource_basic(t *testing.T) {
```

Reply:
done